pkgs/applications/science/logic/isabelle/default.nix

```diff
@@ -1,6 +1,38 @@
-{ lib, stdenv, fetchurl, coreutils, nettools, java, scala, polyml, z3, veriT, vampire, eprover-ho, naproche, rlwrap, perl, makeDesktopItem, isabelle-components, isabelle, symlinkJoin }:
+{ lib, stdenv, fetchurl, coreutils, nettools, java, scala, polyml, z3, veriT, vampire, eprover-ho, naproche, rlwrap, perl, makeDesktopItem, isabelle-components, isabelle, symlinkJoin, fetchhg, runCommand }:
 # nettools needed for hostname
 
+let
+  sha1 = stdenv.mkDerivation {
+    name = "isabelle-sha1";
```

Comment:
Please use pname+version.

Reply:
Not sure what version to put, since I'm fetching from a specific git commit.

Reply:
I've decided to specify the version as `2021-1`.